vhdl_lang/src/analysis/tests/declarations.rs

```diff
@@ -100,3 +100,96 @@ end entity test;
         )],
     )
 }
+
+#[test]
+pub fn disconnect_no_analyze_error() {
+    let mut builder = LibraryBuilder::new();
+    let _ = builder.code(
+        "libname",
+        "\
+entity ent is
+end entity;
+architecture arch of ent is
+  constant foo : time := 10 ns;
+  disconnect all : integer after foo;
+begin
+end arch;
+    ",
+    );
+    check_diagnostics(builder.analyze(), vec![])
```

Comment:
This could be simplified by using `check_no_diagnostics(builder.analyze())`

Reply:
Done